package.json

```diff
@@ -17,11 +17,11 @@
   },
   "scripts": {
     "start": "yarn install && yarn generate && cross-env TS_NODE_PROJECT=\"webpack.tsconfig.json\" && yarn run start:dev",
-    "start:cloud": "yarn install && yarn generate && cross-env TS_NODE_PROJECT=\"webpack.tsconfig.json\" && yarn run start:dev-cloud",
     "start:dev": "webpack-dev-server --config ./webpack.dev.ts",
-    "start:dev-cloud": "cross-env CLOUD_LOGOUT_URL=http://localhost:8080/api/v2/signout CLOUD_URL=http://localhost:4000 webpack-dev-server --config ./webpack.dev.ts",
+    "start:dev:oss": "yarn start:dev",
+    "start:dev:remocal": "NODE_ENV=development UI_SHA=local_dev_mode BASE_PATH=/ API_BASE_PATH=/ PORT=8080 PUBLIC=${PUBLIC:-https://twodotoh.a.influxcloud.dev.local/} CLOUD_URL=/auth yarn start",
+    "start:kind": "echo 'please start using `yarn start:dev:remocal`' && yarn start:dev:remocal",
```

Comment:
Well done on keeping `start:kind` with a message and actually calling `start:dev:remocal`! 

I saw three places in k8s-idpe are using `start:kind`. Just wondering if there are any plans to update them in case `start:kind` is removed or changed to something else?

https://github.com/influxdata/k8s-idpe/blob/0cc3ae11e1ce81d3e793df9ca0aaf831cc6667ba/config/base/base.libsonnet#L1644

https://github.com/influxdata/k8s-idpe/blob/0cc3ae11e1ce81d3e793df9ca0aaf831cc6667ba/scripts/local-dev/ui.bash#L24

https://github.com/influxdata/k8s-idpe/blob/0cc3ae11e1ce81d3e793df9ca0aaf831cc6667ba/scripts/remocal-dev/dev.bash#L155

https://github.com/influxdata/k8s-idpe/blob/0cc3ae11e1ce81d3e793df9ca0aaf831cc6667ba/scripts/remocal-dev/dev.bash#L321

Reply:
No plans yet. We can do that next (after these PRs merge) -- the main goal here was to scope down the minimum needed for proving that the monitor-ci code-axing will not break anything.

Reply:
Got it! That makes sense